### PR TITLE
Update ogh announces to use C_color

### PR DESCRIPTION
**Pull request by Asheraf**
*now announce commands use constant colors instead of hexadecimal.
*add a missing space in wh/kh cards seller

---
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1381)

### CODE REVIEW

File: npc/re/instances/OldGlastHeim.txt [hunk 1]
```diff
@@ -337,7 +337,7 @@ OnEnable:
 	donpcevent(instance_npcname("Heinrich#ghinstance2")+"::OnEnable");
```
Comment (Ridley8819) [inline npc/re/instances/OldGlastHeim.txt]: line 137 0x00ff99
maybe C_LIME?

**Comment by Asheraf** [comment]
lime is 0x00FF00 .. I can't find a name for this color :D

**Comment by Asheraf** [comment]
based on http://chir.ag/projects/name-that-color/#00FF99
it's name is Spring Green so maybe C_SPRINGGREEN ?

**Comment by MishimaHaruna** [comment]
Side note, our C_LIME isn't actually lime, but just plain green, sigh (and that's not the only C_ constant that's a bit off)

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/herculesws/hercules/1381#-:-KO-yKs-PD-g4LODf8NJ:-684232556)*

**Comment by Asheraf** [comment]
@MishimaHaruna then should i set it to C_LIME ?

**Comment by MishimaHaruna** [comment]
I think for now keep it as it is, but we should consider (in a separate pull request or commit), a review of the C_ constants, to add all the values used by official scripts at least.

If you want to add a new C_SPRINGGREEN constant and use it, feel free to

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/herculesws/hercules/1381#-:-KO05gD2frGu0LhnQBIi:988553031)*

**Comment by Asheraf** [comment]
@MishimaHaruna this is done i guess ^^

**Pull request closed without merging**